services/payments/src/simcore_service_payments/api/rest/routes.py

```diff
@@ -1,14 +1,18 @@
-from fastapi import APIRouter, FastAPI
+from fastapi import APIRouter, FastAPI, HTTPException
 
 from ..._meta import API_VTAG
 from . import _acknowledgements, _auth, _health, _meta
+from ._exceptions import handle_errors_as_500, http_exception_as_json_response
 
 
-def setup_rest_api_routes(app: FastAPI):
+def setup_rest_api(app: FastAPI):
     app.include_router(_health.router)
 
     api_router = APIRouter(prefix=f"/{API_VTAG}")
     api_router.include_router(_auth.router, tags=["auth"])
     api_router.include_router(_meta.router, tags=["meta"])
     api_router.include_router(_acknowledgements.router, tags=["acks"])
     app.include_router(api_router)
+
+    app.add_exception_handler(Exception, handle_errors_as_500)
```

Comment:
Q: Last time we were facing a problem with just pure `Exception` during debugging -> that it just raised an exception and the exception handler was not called, this might be still the problem here, correct?

Reply:
@matusdrobuliak66 the problem was how we were interpreting exception handlers. I found in the doc from startlette this https://www.starlette.io/exceptions/. I added some tests as well.

Reply:
Ok thanks!